simphony/io/h5_cuds.py

```diff
@@ -112,6 +113,45 @@ def add_particle_container(self, particle_container):
         self._file.flush()
         return pc
 
+    def add_mesh(self, name, mesh=None):
```

Comment:
I guess related to that, one will also need to handle any changes to `name'.  For example,
```
w_mesh_foo = wrapper.get_mesh("foo")
w_mesh_foo.name = "bar"
bar = wrapper.get_mesh("bar") # should work
foo = wrapper.get_mesh("foo") # should fail
```